google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplTest.java

```diff
@@ -3120,4 +3120,50 @@ public void testV4PostPolicy() {
     assertEquals(outputFields.get("key"), "my-object");
     assertEquals("https://storage.googleapis.com/my-bucket/", policy.getUrl());
   }
+
+  @Test
```

Comment:
new tests are required in mockito, could you rewrite this test and put it to StorageImplMockitoTest

Reply:
Done